cmake/ConfigureBenchmarks.cmake

```diff
@@ -7,4 +7,36 @@ function(configure_benchmark_executable EXECUTABLE_NAME)
     target_include_directories(${EXECUTABLE_NAME} PRIVATE ${PROJECT_INCLUDE_FOLDER})
     target_link_libraries(${EXECUTABLE_NAME} PRIVATE ${LIBRARY_NAME})
 
+    # Link google benchmarks
+    link_libraries("-lbenchmark")
+
+    # Include boost
```

Comment:
Couldn't you just `configure_executable(${EXECUTABLE NAME})` instead of copy-pasting the code?